prioritizer/prioritizer.py

```diff
@@ -5,14 +5,16 @@
 import re
 import json
 import pandas as pd
+from pydantic import BaseModel
+from typing import List
 
```

Comment:
I thought you are using Path and pathlib? :p 
That might be a good thing to change the pathes using `Path`